gatorgrade/output/output_functions.py

```diff
@@ -1,2 +1,31 @@
 # This class is used for storing the main functions requested from the Github Issue Tracker for the output team.

```

Comment:
You used the terminology 'class', but there is no class in this file. Could you use terminology such as 'module' instead to be more clear and prevent misunderstanding?